src/generator/exercises/hello_world.cr

```diff
@@ -8,17 +8,21 @@ class HelloWorldGenerator < ExerciseGenerator
 
   def test_cases
     JSON.parse(data)["cases"].map do |test_case|
-      HelloWorldTestCase.from_json(test_case.to_json)
+      HelloWorldTestCase.new(test_case)
     end
   end
 end
 
 class HelloWorldTestCase < ExerciseTestCase
-  JSON.mapping(
-    description: String,
-    name: String | Nil,
-    expected: String
-  )
+  private getter description : JSON::Any
```

Comment:
The one thing that I'm a bit confused about with are these private getters where you expose one of them (`description`) anyways.

Is it just a personal preference of less public methods plus the preference of private getter methods over accessing instance variables?